File: codigos_alunos/questao9/python/erro_atalho.py
inscritos_confirmados = 120
capacidade_auditorio = 80
cadeiras_extras_disponiveis = 20

capacidade_total = capacidade_auditorio + cadeiras_extras_disponiveis

pessoas_de_fora = max(0, inscritos_confirmados - capacidade_total)

if pessoas_de_fora > 0:
    print("Evento lotado. Pessoas na lista de espera: ", pessoas_de_fora)
else:
    print("Lotação controlada. Evento pronto para iniciar")

File: codigos_alunos/questao9/java/ErroAtalho.java
package Questao9.java;

public class ErroAtalho {
    public static void main(String[] args) {
        int inscritos_confirmados = 120;
        int capacidade_auditorio = 80;
        int cadeiras_extras_disponiveis = 20;
        
        int capacidade_total = capacidade_auditorio + cadeiras_extras_disponiveis;

        int pessoas_de_fora = Math.max(0, inscritos_confirmados - capacidade_total);
        
        if (pessoas_de_fora > 0) {
            System.out.println("Evento lotado. Pessoas na lista de espera: " + pessoas_de_fora);
        } else {
            System.out.println("Lotação controlada. Evento pronto para iniciar");
        }
    }
}
